docs/conf.py

```diff
@@ -302,12 +302,12 @@ def setup(app):
     "topic-guides/user-interfaces": "../interfaces/commands-and-shells.html",
     "topic-guides/users": "../cluster-setup-guide/users.html",
     "topic-guides/yaml": "../introduction.html",
-    "training-apis/api-core/checkpoints": "../../training/apis-howto/api-core/checkpoints.html",
-    "training-apis/api-core/distributed": "../../training/apis-howto/api-core/distributed.html",
-    "training-apis/api-core/getting-started": "../../training/apis-howto/api-core/getting-started.html",
-    "training-apis/api-core/hpsearch": "../../training/apis-howto/api-core/hpsearch.html",
-    "training-apis/api-core/index": "../../training/apis-howto/api-core/overview.html",
-    "training-apis/api-core/metrics": "../../training/apis-howto/api-core/metrics.html",
+    "training-apis/api-core/checkpoints": "../../training/apis-howto/api-core-ug.html",
+    "training-apis/api-core/distributed": "../../training/apis-howto/api-core-ug.html",
+    "training-apis/api-core/getting-started": "../../training/apis-howto/api-core-ug.html",
+    "training-apis/api-core/hpsearch": "../../training/apis-howto/api-core-ug.html",
+    "training-apis/api-core/index": "../../training/apis-howto/api-core-ug.html",
+    "training-apis/api-core/metrics": "../../training/apis-howto/api-core-ug.html",
     "training-apis/api-core/reference": "../../reference/reference-training/training/api-core-reference.html",
```

Comment:
last thing, can we also add links from the documents you are deleting to the new consolidated document?

Reply:
more info needed

Reply:
thank you